src/pages/OrderBook.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect } from 'react'
+import { ContentPage } from 'components/Layout/PageWrapper'
+import OrderBookWidget from 'components/OrderBookWidget'
+import TokenSelector from 'components/TokenSelector'
+import { useTokenList } from 'hooks/useTokenList'
+import { useWalletConnection } from 'hooks/useWalletConnection'
+import { Network } from 'types'
+import useSafeState from 'hooks/useSafeState'
+import { TokenDetails } from 'types'
+
+const OrderBook: React.FC = () => {
+  const { networkId } = useWalletConnection()
+  const fallBackNetworkId = networkId ? networkId : Network.Mainnet // fallback to mainnet
```

Comment:
Do we need to show prices, or even, does the tool have prices for anything that's not Mainnet?

Reply:
First version Felix did was using pure web3, so only that was needed (that, and the batchexchange contract and the viewer)

I think he is using now his own service, and I believe he also has two environments (rinkeby and mainnet), so probably yes, I think we should show the current network, and mainnet as a default

Reply:
Yes, I'm querying the backend service and need network info to form the correct URL